core/templates/dev/head/editor/feedback_tab/ThreadDataService.js

```diff
@@ -133,8 +153,12 @@ oppia.factory('threadDataService', [
       if (commitMsg) {
         payload.commit_message = commitMsg;
       }
-      $http.put(_SUGGESTION_ACTION_HANDLER_URL + threadId, payload).success(
-        onSuccess).error(onFailure);
+      $http.put(_SUGGESTION_ACTION_HANDLER_URL + threadId, payload).success(function() {
```

Comment:
Note that resolving suggestions is also going to lead to changes in the thread count.